treeherder/perf/models.py

```diff
@@ -450,6 +450,26 @@ class PerformanceAlert(models.Model):
 
     manually_created = models.BooleanField(default=False)
 
+    @property
+    def initial_culprit_job(self) -> Optional[Job]:
+        if hasattr(self, '__initial_culprit_job'):
+            return self.__initial_culprit_job
+
+        try:
+            culprit_data_point = (  # original one, not necessarily the real culprit' s one
```

Comment:
Can you add more context to this comment please.

Reply:
To get the culprit data point we filter by "summary.push", so we will inherit the same concern as there:

"An alert is always a member of an alert summary, which groups all
    the alerts associated with a particular push together. In many cases at
    Mozilla, the original alert summary is not correct, so we allow reassigning
    it to a different (revised) summary."